File: proj10/java/Ostrich.java
/**
 * Ostrich.java defines the behavior of an ostrich.
 *
 * Completed by: David Reidsma
 * Date: 04/30/2020
 */

public class Ostrich extends WalkingBird {
    /**
     * implicit constructor
     * Postcondition: myName = ""
     */
    public Ostrich() {
	super();
    }

    /**
     * explicit constructor
     * @param name a String 
     * PostCond: myName == name.
     */
    public Ostrich(String name) {
	super(name);
    }

    /**
     * Get this Bird's call.
     * @return The call of the bird.
     */
    public String getCall() {
	return "Rrr Rrr Rrrrrrr!";
    }
}

File: proj10/ruby/ostrich.rb
# ostrich.rb defines the behavior of an ostrich.
# Author: David Reidsma
# Date: 04/30/2020

require './walking_bird.rb'

class Ostrich < WalkingBird
  # Get the call of this bird.
  def call
    return "Rrr Rrr Rrrrrr!"
  end
end
